src/main/java/com/ea/restaurant/controller/UserController.java

```diff
@@ -0,0 +1,60 @@
+package com.ea.restaurant.controller;
+
+import com.ea.restaurant.entities.User;
+import com.ea.restaurant.service.impl.UserServiceImpl;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
```

Comment:
Add RequestMapping for `/users` here that way you don't need to add `users` in all paths

Reply:
Done.